holmes/plugins/sources/prometheus/plugin.py

```diff
@@ -72,6 +74,22 @@ def fetch_issues(self) -> List[Issue]:
             )
             for alert in alerts
         ]
+        return self.label_filter_issues(issues)
+
+    def label_filter_issues(self, issues: List[Issue]) -> List[Issue]: 
+        if not self.label:
+            return issues
+
+        label_parts = self.label.split('=')
+        if len(label_parts) != 2:
+            raise Exception(f"The label {self.label} is of the wrong format use '--alertmanager-label key=value'")
+
+        alert_label_key, alert_label_value = label_parts
+        filtered_issues = [issue for issue in issues if issue.raw.get("labels", {}).get(alert_label_key, None) == alert_label_value]
+        if not filtered_issues:
+            raise Exception(f"No valid alerts with the label {alert_label_key} and value {alert_label_value}")
```

Comment:
Please make the behaviour consistent with alertname filtering. (If there are no matching alerts we return an empty list instead of throwing an exception.)